logging/connection_tracer.go

```diff
@@ -34,6 +34,7 @@ type ConnectionTracer struct {
 	LossTimerExpired                 func(TimerType, EncryptionLevel)
 	LossTimerCanceled                func()
 	ECNStateUpdated                  func(state ECNState, trigger ECNStateTrigger)
+	ChoseAlpn                        func(protocol string)
```

Comment:
I think Go naming conventions would tell us to write it this way:
```suggestion
	ChoseALPN                        func(protocol string)
```